modules/virt-enlarging-vm-disk.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-vm-disk-enlargement.adoc
+
+[id="virt-enlarging-vm-disk_{context}"]
+= Enlarging a virtual machine disk
+
+A virtual machine (VM) disk has a default or initial storage capacity when you create it. However, the disk can be enlarged by resizing the disk's persistent volume claim (PVC).
+
+VM disk enlargement is disabled by default because an operating system may have built-in caching of disk writes. To expand a file system, you need to write to portions of the disk that are already in use. Doing so may result in corrupt data, unless the operating system expects this kind of operation to happen.
```

Comment:
I am confused by this paragraph.  From my understanding, resize is disabled by default because not everyone will want this implicit behavior.  I do anticipate this being defaulted to enabled in a future release.

I am not aware of any corruption risks associated with this feature but would be interested to see the source of this.

Reply:
Also, how do I enable it?

Reply:
To follow up on the comment about the confusing paragraph, my source was an upstream doc PR that Maya Rashish gave me - https://github.com/kubevirt/kubevirt/blob/6dd1fcb1ec1b7eed60179da9214919089dd5ef4f/docs/disk-expansion.md

There is a section on "Why we don't expand file systems" that mentions the information I included in that paragraph. Hopefully it makes sense. It's also possible that I've misinterpreted the upstream info, leading to the confusion about the paragraph. If so, please feel free to correct me.

As for the how to enable it, the procedure is right there in the section. I followed the steps that Maya indicated, including the YAML example.

Reply:
Yes, we don't need to mention that detail and scare customers.  Instead, we should say something like:

This feature makes the extra space available to the virtual machine but it is the responsibility of the VM owner to decide how to consume the storage. 

We could also link to documentation on how to do it in Windows and RHEL:
https://docs.microsoft.com/en-us/windows-server/storage/disk-management/extend-a-basic-volume
https://access.redhat.com/solutions/29095
https://access.redhat.com/solutions/24770

Reply:
I've removed the confusing lines and replaced them with the suggested explanation.